### PR TITLE
docs: minor doc fixes of showing in progress clones for a snapshot

### PR DESCRIPTION
**Pull request by nmshelke**
Signed-off-by: Nikhilkumar Shelke <nshelke@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/tasks/cephfs/test_volumes.py [hunk 1]
```diff
@@ -5297,7 +5297,7 @@ def test_clone_subvolume_info(self):
 
     def test_subvolume_snapshot_info_without_snapshot_clone(self):
         """
-        Verify subvolume snapshot info output without clonnnig snapshot.
+        Verify subvolume snapshot info output without clonig snapshot.
```
Comment (neesingh-rh) [inline qa/tasks/cephfs/test_volumes.py:5300]: s/clonig/cloning

**Review by anthonyeleven — APPROVED** [review]

**Review by rishabh-d-dave — APPROVED** [review]

**Comment by nmshelke** [comment]
jenkins test windows

**Review by kotreshhr — APPROVED** [review]

**Comment by dparmar18** [comment]
typo in the title of 2nd commit, looks good otherwise

**Comment by tchaikov** [comment]
this is document-only change. i don't think we need to run it against any qa suite.

**Comment by zdover23** [comment]
@nmshelke : squash these two commits into a single commit and I'll merge this PR.

If you need squashing instructions, let me know.

**Comment by nmshelke** [comment]
> @nmshelke : squash these two commits into a single commit and I'll merge this PR.
> 
> If you need squashing instructions, let me know.

@zdover23 Please check. I have squashed commits.

**Review by dparmar18 — APPROVED** [review]

**Comment by nmshelke** [comment]
> typo in the title of 2nd commit, looks good otherwise

Squashed commits. Hence no message exist for second commit

**Review by vshankar — APPROVED** [review]

**Review by anthonyeleven — APPROVED** [review]

**Review by ajarr — APPROVED** [review]

**Pull request merged**